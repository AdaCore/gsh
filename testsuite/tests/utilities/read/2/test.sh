target="x86_64-pc-mingw32"
field1="a"
IFS="-" read -r field1 field2 field3 field4 <<EOF
$target
EOF
echo IFS=$IFS            # test unset variable before the read
echo field1 = ${field1}  # pre-existing variable but must have change
echo field2 = $field2    # variable created during the read
echo field3 = $field3    # ditto
echo field4 = $field4    # ditto
