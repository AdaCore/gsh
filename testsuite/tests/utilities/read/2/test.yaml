title: read builtins, -r option + command scope level
