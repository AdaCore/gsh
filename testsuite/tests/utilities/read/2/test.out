[warning] read: -r is currently ignored
IFS=
field1 = x86_64
field2 = pc
field3 = mingw32
field4 =
